src/nupic/utils/SdrMetrics.hpp

```diff
@@ -200,6 +200,19 @@ class ActivationFrequency : public MetricsHelper_ {
      */
     ActivationFrequency( const std::vector<UInt> dimensions, UInt period );
 
+    /**
+     * Assign an initial activation frequency to all bits in the SDR.  If this
+     * method is used then this metric will always use the exponential moving
+     * average, and skip over the regular average which is usually applied to
```

Comment:
I don't like the fact that the name suggests "initialize to a value", while it enforces a computation mode (EMA vs normal average). 
Can you make it an argument? `initialize(Real initValue, bool=useExponentialMovingAverage=false)` ?

Reply:
No, because the argument initialValue requires that it always use the exponential rolling average, and never use the regular average.  The regular average will discard the initial value.

Reply:
nit, then maybe name initializeExponentialMovingAvg()?